BUILD.bazel

```diff
@@ -7,16 +7,17 @@ exports_files([
 ])
 
 filegroup(
```

Comment:
note, you might want to convert this to a `node_modules_filegroup` see https://github.com/bazelbuild/rules_nodejs/blob/master/defs.bzl#L41

Reply:
Changed. Thanks for the tip. 👍